src/components/ActionBar/scss/ActionBar.scss

```diff
@@ -0,0 +1,31 @@
+
+/*------------------------------------*\
+    $ACTION BAR
+\*------------------------------------*/
+
+@import '../../../scss/index';
+
+.action-bar {
+  @include menu-anchor-collapse();
+  @include link--transition();
+  @include soft--sides;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  background-color: black;
```

Comment:
i don't think this should be a style you need to override

Reply:
sure, but then that could be said more or less for the others also.... where do we stop?

Reply:
Similar to my other comment- this moves away from "functional" styles and into stylistic decisions.